### ISSUE TITLE
Improve i3pyblocks.types.Dictable

### ISSUE THREAD

**Issue by thiagokokada** [thiagokokada/i3pyblocks#22]
`i3pyblocks.types.Dictable` is very useful, but for now it is kind limited since it is always result in a Dict from `Any` to `Any`.

Would be nice to allow `Dictable` to be more precise, for example:

```python
foo: Dictable[str, bool] = {1: null}  # Error
```

**Comment by thiagokokada** [comment]
If not possible, something like this would be already nice:

```python
DictableStrBool = Dictable(str, bool)
foo: DictableStrBool = {1: null}  # Error
```

**Comment by thiagokokada** [comment]
`TypedDict` is the proper solution for this problem, but it is only available in Python 3.8: https://docs.python.org/3/library/typing.html#typing.TypedDict

**Comment by thiagokokada** [comment]
Closed by: #26

**Comment by thiagokokada** [comment]
Closed by: #26
